modules/installation/pages/hardware-requirements.adoc

```diff
@@ -52,5 +54,19 @@ For {smr} hardware requirements, see xref:retail:retail-requirements.adoc[].
 |                          | _Production Server_ Minimum 8{nbsp}GB
 | Disk Space:              | [path]``/`` _(root)_ Minimum 24{nbsp}GB
 |                          | [path]``/srv`` Minimum 100{nbsp}GB
-|                          | [path]``/var/cache`` _(Squid)_ Minimum 100{nbsp}GB
+|                          | [path]``/var/cache`` _(Squid)_ Recommended: minimum 100{nbsp}GB
 |===
+
+{productname} Server stores packages in the [path]``/var/spacewalk/`` directory. 
+Repository synchronization fails if this directory runs out of disk space. 
+You can estimate how much space the [path]``/var/spacewalk/`` directory requires based on the number and type of clients and repositories you plan to mirror.
+
+{productname} Proxy caches packages in the [path]``/var/cache/`` directory.
+If there is not enough space available in [path]``/var/cache/``, the proxy will remove old, unused packages and replace them with newer packages.
+
+As a result of this behavior:
+
+* The larger [path]``/var/cache/`` directory is on the {productname} Proxy, the less traffic there will be between it and the {productname} Server. 
+* By making the [path]``/var/cache/`` directory on the {productname} Proxy at least the same size as [path]``/var/spacewalk/`` in the {productname} Server, you avoid a large amount of traffic after the first synchronization.
```

Comment:
"at least the same size" - Makeing it bigger is totally useless. So I would suggest we remove "at least".
Even same size is not recommended/needed. You typically never use all packages behind the proxy. So you would just spend never used disk space. But this is a decission of the admin.